### ISSUE TITLE
Replace the static font buttons with a font select menu

### ISSUE THREAD

**Issue by Delta-official** [nuttyartist/notes#610]
I'm liking Notes so far, but the inability to use a custom font is a problem for me as I don't like any of the fonts included. Perhaps there could be a dropdown or some sort of menu that allows you to select one of the system fonts?

**Comment by nuttyartist** [comment]
I agree that on Linux it can be quite limiting since there are not many options. We might create an advanced preferences window to allow adding more fonts.

**Comment by Delta-official** [comment]
will it limit the fonts to sans in kanban mode? (sidenote: why is it even restricted in kanban mode? it doesn't make much sense to me)

**Comment by nuttyartist** [comment]
That's a good question. I'm not sure I have a good answer apart than that's what made sense at the time I developed it. Maybe we should change it for the next version. I'll create a new issue for that.

**Comment by whdmike** [comment]
Just would like to know how is a situation in this area. Personally, I like a Noto Sans font, and I am missing that option. There are three styles options, but I think it might be easier just to choose a font of your choice.

**Comment by nuttyartist** [comment]
@whdmike I agree. When we'll implement a proper settings window we'll allow changing the font to a custom one.

**Comment by MiPe55** [comment]
Interestingly, I started to use Notes six months ago, and stopped, because I wanted to use Noto Sans font, which I prefer to use for my articles. Now, after a while I wanted to give the app another chance, because Iotas that I am using at the moment have a strange way to listing notes. And again, font is almost a dealbreaker. I came here to open a ticket, because I forgot I commented already six months ago. I am writing just to remind that we are here, and we like the app a lot, but we would like to like it even more ... Thanks.

PS. Before, I commented under "whdmike" - I had another github account.

**Comment by nuttyartist** [comment]
@MiPe55 I hear you! Currently, I don't have a lot of time to put into the Notes, since my day job at a startup is very intense. But it's definitely on the bucket list to implement this.
